workspaces/ui-v2/src/components/sharing/ShareButton.tsx

```diff
@@ -57,19 +62,22 @@ export const ShareButton: React.FC<{}> = (props) => {
       const token = await getAccessTokenSilently();
 
       let newSpecResp = await fetch(
-        `${baseDomain}/api/person/public-specs-v2`,
+        `${baseDomain}/api/person/public-specs-v3`,
         {
           method: 'POST',
```

Comment:
This is also something that could be moved into a `/ui-v2/src/clients` folder similar to the cli-client client